concourse/build.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+ARGV_DIRECTORY="$1"
+
+pushd ${ARGV_DIRECTORY}
+
+# Install dependencies for build and deploy scripts
+npm install
+pip install -r requirements.txt
+
+# HACK: Override fly version coming from the image with a newer one that works with our concourse config
+wget -qO- https://github.com/concourse/concourse/releases/download/v5.3.0/fly-5.3.0-linux-amd64.tgz \
```

Comment:
@stathismor you can make this easier and grab the correct version from `https://ci.balena-dev.com/api/v1/cli?arch=amd64&platform=linux`

Reply:
:exploding_head: I didn't know that, cheers, done!